Add OKLCH and OKLab functions to reference

ai opened the pull request:
<!-- 👀 Thanks for opening a PR! Read comments like this one to get your PR merged faster. -->
#### Summary
<!-- ✍️ In a sentence or two, describe your changes -->

I added OKLab color functions to [CSS Functional Notation](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Functions).

#### Motivation
<!-- ❓ Why are you making this change? Help us understand how your changes help readers. -->

They are part of [CSS Color Modules 4](https://www.w3.org/TR/css-color-4/).

Adding them to this list will fix Stylelint [`function-no-unknown`](https://stylelint.io/user-guide/rules/list/function-no-unknown/) since Stylelint uses this document as a source of all valid CSS function names.

#### Supporting details
<!-- 🔗 Link to supporting information, such as bug trackers, source control, release notes, and vendor docs. -->

[Safari TP](https://developer.apple.com/safari/technology-preview/release-notes/) should support these functions.

#### Metadata
<!-- ✅ Check a box if applicable, like this: [x]

This PR…
-->
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

<!-- 👷‍♀️ After submitting, review the results of the "Checks" tab! -->

ai committed:
Add OKLCH and OKLab functions to reference

teoli2003 commented:
Is there any browser implementing them? If not, we should wait before merging this.

ai commented:
> Is there any browser implementing them?

Safari TP mentions `oklch()` support in [release notes](https://developer.apple.com/safari/technology-preview/release-notes/).

teoli2003 commented:
Assigning myself for review (I'm reorganizing CSS Colors). Before landing this, I want to be sure the BCD information is up-to-date, and the subpages created (the mention is pretty useless without them)

ai committed:
Add oklab() function page

ai committed:
Copy lab redirects for oklab

ai committed:
Add oklch() page

ai commented:
@teoli2003 I added pages for `oklab` and `oklch`

I took `lch()` and `lab()` pages and:

1. Replaced names
2. Replaced examples according to [OKLCH Converter](https://oklch.evilmartians.io/) (`oklab()` result was converted by `culori` Node.js module)
3. Replaced Safari TP release
4. Replaced link to Oklab announce article
5. Replaced `CIE Lightness` to `perceived lightness` (the creator ok Oklab used this term).
6. Replaced `C` max value to max value for sRGB and P3 colors (Rec. 2020 `C` value is `0.5` but there is no Rec. 2020 monitors yet).

I also added `oklch()` and `oklab()` to redirects.

Questions:

1. How can I add compatibility data?
2. Will `{{Specifications}}` macros work automatically?

ai commented:
@teoli2003 just to clarity why I force these changes:

The reason of this PR is not to add docs but to solve problem with CSS linters. They use MDN as a source of valid functions and right now, any CSS code with `oklch()` will be marked as invalid.

OnkarRuikar commented on files/en-us/web/css/color_value/oklch/index.md:
```diff
@@ -0,0 +1,49 @@
+  - Reference
+  - Web
+  - color
+  - oklch
```
```suggestion
  - oklch
  - Experimental
```

OnkarRuikar commented on files/en-us/web/css/color_value/oklab/index.md:
```diff
@@ -0,0 +1,50 @@
+  - Reference
+  - Web
+  - color
+  - oklab
```
```suggestion
  - oklab
  - Experimental
```

teoli2003 commented:
Hi, ai! No worry, I just didn't have enough time to go back to this. I will review this PR this week!

ai committed:
Apply suggestions from code review

Co-authored-by: Onkar Ruikar <87750369+OnkarRuikar@users.noreply.github.com>

ai commented:
@teoli2003 friendly ping for the glory of OKLCH 😺

teoli2003 reviewed (approved):
👍

ai commented:
Thanks! Just a small question: will it deploy automatically or did I need another step to make it public?

https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/oklch

teoli2003 commented:
It will deploy automatically, in about 24h.

ai commented:
I sent PR to compat data as well https://github.com/mdn/browser-compat-data/pull/16526
